src/i_sound.c

```diff
@@ -499,11 +382,26 @@ int I_StartSound(sfxinfo_t *sound, int vol, int sep, int pitch, boolean loop)
   if (channel == MAX_CHANNELS)
     return -1;
 
-  if (CacheSound(sound, channel, pitch))
+  if (CacheSound(sound, channel))
   {
+    ALuint source = openal_sources[channel];
+    ALuint buffer = *channelinfo[channel].data;
+
     channelinfo[channel].idnum = id++; // give the sound a unique id
-    Mix_PlayChannelTimed(channel, &channelinfo[channel].chunk, loop ? -1 : 0, -1);
     I_UpdateSoundParams(channel, vol, sep);
+
+    alSourcei(source, AL_BUFFER, buffer);
+    alSourcei(source, AL_LOOPING, loop ? AL_TRUE : AL_FALSE);
+    if (pitch != NORM_PITCH)
+    {
+      alSourcef(source, AL_PITCH, steptable[pitch]);
```

Comment:
Will we have to reset this at some point? It currently applies to `source` which is `openal_sources[channel]`, so this is a property set for that specific channel. And if pitch-shifting is disabled after a channel has played a pitch-shifted sound, will this property remain?

Reply:
> will this property remain?

It seems not, I turn off this option in the menu and it is turned off immediately.